packages/react-app-revamp/components/_pages/ProposalContent/index.tsx

```diff
@@ -64,24 +63,21 @@ const ProposalContent: FC<ProposalContentProps> = ({ proposal }) => {
 
   return (
     <div className="flex flex-col w-full h-96 md:h-80 animate-appear rounded-[10px] border border-neutral-11 hover:bg-neutral-1 cursor-pointer transition-colors duration-500 ease-in-out">
-      <ProposalContentInfo
-        authorAddress={proposal.authorEthereumAddress}
-        rank={proposal.rank}
-        isTied={proposal.isTied}
-        isMobile={isMobile}
-        commentsCount={proposal.commentsCount}
-      />
-      <Link
-        href={`/contest/${chainName}/${contestAddress}/submission/${proposal.id}`}
-        shallow
-        scroll={false}
-        className="flex items-center overflow-hidden px-14 h-80 md:h-72"
-      >
-        <Interweave
-          className="line-clamp-6 md:line-clamp-4 markdown overflow-y-hidden text-[16px] md:text-[18px]"
-          content={truncatedContent}
-          transform={transform}
+      <Link href={`/contest/${chainName}/${contestAddress}/submission/${proposal.id}`} shallow scroll={false}>
```

Comment:
Oo question here: how come scroll is false? Could that be the issue with #1195?

Reply:
Hm that doesn't relate with #1195, as scrolling there is primarily done to determine the position of the next page. i believe i passed it as `false` for a reason, but i need to recall precisely why.

Reply:
Ah gotcha gotcha, ok just thought I'd check!